### PR TITLE
Feature update spanish home and pages

### PR DESCRIPTION
**Pull request by alonsolopezr**
**Description**
I'm updating the docs home for español lang, to be displayed in the same layout, as the english page. Also, updating several pages and it's link in the home.

This PR fixes #

**Notes for Reviewers**
Please check this PR @jesuslerma, @ramrodo.. thanks

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1e2aa055eec221643d66aea329ea7e74c9cf4ca8 at: https://615a20222a210ff113bd7ad1--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2f9134fc9d1bf916b21881c4f71c956b51031fb3 at: https://615a20ee20489ca88033e708--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@alonsolopezr, heads-up: there is a merge conflict to address in this PR.
// @jesuslerma

**Comment by alonsolopezr** [comment]
Thanks @leecalcote for the review… I will fix this one and look for more of these conflicts that I didn’t notice. Thanks again

File: docs/pages/es/concepts/architecture/architecture.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Arquitectura
+permalink: /es/concepts/architecture
+abstract: "descripción general de los diferentes componentes individuales de la arquitectura Meshery y cómo interactúan como sistema."
```
Comment (ramrodo) [inline docs/pages/es/concepts/architecture/architecture.md:5]:
```suggestion
abstract: "Descripción general de los diferentes componentes individuales de la arquitectura Meshery y cómo interactúan como sistema."

```

File: docs/pages/es/concepts/architecture/architecture.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# Arquitectura
+
+#### La arquitectura de Meshery puede ser vista desde dos perspectivas:
+ 
+##### 1. [**Clientes**](#1-client-architecture)
+##### 2. [**Proveedores**](#2-provider-architecture)
+
+![Meshery architecture](/assets/img/architecture/Meshery-architecture-diagram.png)
+
+### 1. **Arquitectura del Client**
```
Comment (ramrodo) [inline docs/pages/es/concepts/architecture/architecture.md:21]:
```suggestion
### 1. **Arquitectura del Cliente**

```

File: docs/pages/es/extensibility/load-generators.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+lang: es
+categories: es
+list: include
+---
+
+Los usuarios pueden preferir usar un generador de carga sobre el siguiente dada la diferencia de capacidades entre los generadores de carga, por lo que Meshery proporciona una "interfaz de generador de carga" (una interfaz gRPC) detrás de la cual se puede implementar un generador de carga. Meshery ofrece a los usuarios la opción de elegir qué generador de carga prefieren utilizar para una prueba de rendimiento determinada. Los usuarios pueden configurar su propia preferencia de generador de carga diferente al generador de carga predeterminado.
+
+### ¿Qué función cumplen los generadores de carga en Meshery?
+
+Los generadores de carga proporcionarán la capacidad de ejecutar pruebas de carga desde Meshery. A partir de hoy, los generadores de carga están integrados como bibliotecas en Meshery y Meshery invoca las API de los generadores de carga con las opciones de prueba de carga adecuadas para ejecutar la prueba de carga. Por el momento, Meshery tiene soporte para generadores de carga HTTP. El soporte para pruebas de carga de GRPC y TCP está en la hoja de ruta. Meshery tiene integración funcional con fortio, wrk2 y nighthawk.
```
Comment (ramrodo) [inline docs/pages/es/extensibility/load-generators.md:18]:
```suggestion
Los generadores de carga proporcionarán la capacidad de ejecutar pruebas de carga desde Meshery. A partir de hoy, los generadores de carga están integrados como bibliotecas en Meshery y Meshery invoca las API de los generadores de carga con las opciones de prueba de carga adecuadas para ejecutar la prueba de carga. Por el momento, Meshery tiene soporte para generadores de carga HTTP. El soporte para pruebas de carga de gRPC y TCP está en la hoja de ruta. Meshery tiene integración funcional con fortio, wrk2 y nighthawk.

```

File: docs/pages/es/project/CONTRIBUTING-gitflow.md [hunk 4]
```diff
@@ -1,126 +1,126 @@
...
+
+Simplemente dirígete a la página de GitHub y haz clic en el botón "Fork". Es así de simple. Una vez que hayas hecho eso, puedes usar tu cliente git favorito para clonar tu repositorio o simplemente dirigirse directamente a la línea de comando:
+
+## Clona tu fork en tu máquina local
+
+```
+git clone git@github.com:USERNAME/FORKED-PROJECT.git
+```
+
+Mantener Tu Fork Actualizado
```
Comment (ramrodo) [inline docs/pages/es/project/CONTRIBUTING-gitflow.md:19]:
```suggestion
## Mantener Tu Fork Actualizado

```

**Comment by warunicorn19** [comment]
@alonsolopezr please take a look at the merge conflicts.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Wow. What a health set of work that you've completed here, @alonsolopezr!

**Comment by leecalcote** [comment]
@shubham-tyagi will you have a look at this PR, suggest, and execute on next steps? Close PR? Resolve merge conflicts?

**Comment by shubham-tyagi** [comment]
sure @leecalcote, will do that.

**Comment by leecalcote** [comment]
@shubham-tyagi is this PR ready to close?

**Comment by shubham-tyagi** [comment]
@leecalcote I actually cloned this PR into #4651 to be able to solve the merge conflicts and add some more changes.

**Pull request merged**